numpy/distutils/fcompiler/gnu.py

```diff
@@ -256,25 +259,29 @@ def version_match(self, version_string):
             # use -mno-cygwin flag for gfortran when Python is not
             # Cygwin-Python
             if sys.platform == 'win32':
-                for key in ['version_cmd', 'compiler_f77', 'compiler_f90',
-                            'compiler_fix', 'linker_so', 'linker_exe']:
+                for key in [
+                        'version_cmd', 'compiler_f77', 'compiler_f90',
+                        'compiler_fix', 'linker_so', 'linker_exe'
+                ]:
                     self.executables[key].append('-mno-cygwin')
         return v
 
     possible_executables = ['gfortran', 'f95']
     executables = {
-        'version_cmd'  : ["<F90>", "-dumpversion"],
-        'compiler_f77' : [None, "-Wall", "-g", "-ffixed-form",
-                          "-fno-second-underscore"] + _EXTRAFLAGS,
-        'compiler_f90' : [None, "-Wall", "-g",
-                          "-fno-second-underscore"] + _EXTRAFLAGS,
-        'compiler_fix' : [None, "-Wall",  "-g","-ffixed-form",
-                          "-fno-second-underscore"] + _EXTRAFLAGS,
-        'linker_so'    : ["<F90>", "-Wall", "-g"],
-        'archiver'     : ["ar", "-cr"],
-        'ranlib'       : ["ranlib"],
-        'linker_exe'   : [None, "-Wall"]
-        }
+        'version_cmd': ["<F90>", "-dumpversion"],
+        'compiler_f77':
+        [None, "-Wall", "-g", "-ffixed-form", "-fno-second-underscore"
+         ] + _EXTRAFLAGS,
+        'compiler_f90':
+        [None, "-Wall", "-g", "-fno-second-underscore"] + _EXTRAFLAGS,
+        'compiler_fix':
+        [None, "-Wall", "-g", "-ffixed-form", "-fno-second-underscore"
+         ] + _EXTRAFLAGS,
+        'linker_so': ["<F90>", "-Wall", "-g"],
+        'archiver': ["ar", "-cr"],
+        'ranlib': ["ranlib"],
+        'linker_exe': [None, "-Wall"]
```

Comment:
this formatting change is for the worse, the old version was easier to read - can you undo?